Libraries/src/Amazon.Lambda.Annotations/LambdaGenerateMainAttribute.cs

```diff
@@ -0,0 +1,13 @@
+﻿
+using System;
+
+namespace Amazon.Lambda.Annotations
+{
+    /// <summary>
+    /// Indicates this the Lambda function is going to target an executable instead of a class based handler.
```

Comment:
Doc suggestion:
```
Used when deploying Lambda functions as a .NET executable instead of a class library. Ensure the .NET project's output type is configured for `Console Application`. In the project file the `OutputType` property will be set to `Exe`.

Deploying as an executable versus a class library is required when compiling functions with Native AOT. It can also be useful to deploy as an executable to include specific versions of `Amazon.Lambda.RuntimeSupport` the .NET Lambda runtime client.
```